api/envoy/config/core/v3/grpc_service.proto

```diff
@@ -288,10 +288,15 @@ message GrpcService {
   // request.
   google.protobuf.Duration timeout = 3;
 
+  // Indicates the retry policy for re-establishing the gRPC channel
+  // This field is optional. If not provided, base interval is set to 500 ms, max interval set to 30 secs.
+  // If max interval is not provided, it will be set to ten times the base interval.
+  RetryPolicy retry_policy = 5;
+
   // Additional metadata to include in streams initiated to the GrpcService. This can be used for
   // scenarios in which additional ad hoc authorization headers (e.g. ``x-foo-bar: baz-key``) are to
   // be injected. For more information, including details on header value syntax, see the
   // documentation on :ref:`custom request headers
   // <config_http_conn_man_headers_custom_request_headers>`.
-  repeated HeaderValue initial_metadata = 5;
+  repeated HeaderValue initial_metadata = 6;
```

Comment:
Don't renumber proto field, this breaks API backward compatibility.

Reply:
corrected, thanks!